dynamiqs/plots/__init__.py

```diff
@@ -4,12 +4,24 @@
 from .namespace import *
 from .utils import *
 
+# diverging
 cmap_colors = [
     (0.0, '#05527B'),
-    (0.225, '#639DC1'),
+    (0.25, '#639DC1'),
```

Comment:
I had adjusted these values so the map was linearly increasing in lightness

Reply:
Hooo, ok sorry, I'll fix it back.